resources/virt.go

```diff
@@ -49,18 +62,21 @@ type VirtRes struct {
 	Transient  bool               `yaml:"transient"` // defined (false) or undefined (true)
 	CPUs       uint16             `yaml:"cpus"`
 	Memory     uint64             `yaml:"memory"` // in KBytes
+	OSInit     string             `yaml:"osinit"` // init used by lxc
```

Comment:
Do you think there's any way to have this be a more general property that other domain types can use too? IOW, is there some equivalent the kvm and friends might use?

This way we can try to avoid having domain specific parameters as much as possible.